### DISCUSSION TITLE
Change the Text Concatenate to allow the use of any delimiter

### DISCUSSION THREAD

**Discussion by Chubbly** [WASasquatch/was-node-suite-comfyui#156] [Ideas]
The current implementation of Text Concatenate only allows a linebreak between text concatenations. Would it be possible to change that to allow any delimiter instead, e.g., a comma (,), slash (/), etc?

This would be useful for prompt concatenation, filename settings, etc.

**Comment by WASasquatch** [comment]
Yes that's possible, and a good idea. I'll get that in as soon as I can.
